subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/ExecutionResult.java

```diff
@@ -66,6 +66,15 @@
      */
     String getError();
 
+    /**
+     * Retrieves the output line that contains the passed in text.
+     *
+     * Fails with an assertion if not output line contains the given text.
```

Comment:
```suggestion
     * Fails with an assertion if no output line contains the given text.
```

Reply:
What happens when there are multiple lines with the text?

Reply:
You get the first, will update javadoc.